src/components/ContentHighlights/ContentHighlightCardItem.jsx

```diff
@@ -1,36 +1,49 @@
 import React from 'react';
-import { Card } from '@edx/paragon';
+import { Card, Hyperlink } from '@edx/paragon';
 import Truncate from 'react-truncate';
 import PropTypes from 'prop-types';
-import { FOOTER_TEXT_BY_CONTENT_TYPE } from './data/constants';
+import { getContentHighlightCardFooter } from './data/constants';
 
 const ContentHighlightCardItem = ({
+  isLoading,
   title,
+  hyperlink,
   contentType,
   partners,
   cardImageUrl,
+  price,
 }) => {
-  const cardLogoSrc = partners?.length === 1 ? partners[0].logoImageUrl : undefined;
-  const cardLogoAlt = partners?.length === 1 ? `${partners[0].name}'s logo` : undefined;
-  const cardSubtitle = partners?.map(p => p.name).join(', ');
-
+  const cardInfo = {
+    cardTitle: (<Truncate lines={3} title={title}>{title}</Truncate>),
+    cardLogoAlt: partners?.length === 1 ? `${partners[0].name}'s logo` : undefined,
+    cardLogoSrc: partners?.length === 1 ? partners[0].logoImageUrl : undefined,
+    cardSubtitle: partners?.map(p => p.name).join(', '),
+    cardFooter: getContentHighlightCardFooter(price, contentType),
+  };
+  if (hyperlink) {
+    cardInfo.cardTitle = (
+      <Hyperlink destination={hyperlink} target="_blank">
+        <Truncate lines={3} title={title}>{title}</Truncate>
+      </Hyperlink>
```

Comment:
Hm, not sure a way around this, but it looks like the external link icon wraps onto a 4th line, since the text truncation doesn't consider the `Hyperlink` icon. And it looks like we can't have `Truncate` wrap `Hyperlink` instead.

<img width="1461" alt="image" src="https://user-images.githubusercontent.com/2828721/208136737-4f534728-4f70-48b0-acc4-9f72a9d76800.png">

I think for the purpose of this PR we should leave it as you have it, but let's confirm with Alena if this is OK. I personally feel having the external link icon sometimes wrap to a 4th line doesn't look all that clean and would possibly prefer a different approach to indicating opening in a new tab, e.g.:

<img width="301" alt="image" src="https://user-images.githubusercontent.com/2828721/208137352-19655ca2-509e-433e-8639-e84bf5c2f799.png">

Of course, let's defer to the decision is by Alena.

Reply:
Leaving open for future reference